### ISSUE TITLE
Task 1 file 

### ISSUE THREAD

**Issue by pratibha77118** [ubsuny/23-Homework5G2#62]
Hi all,
can you check on my file to see if it is working fine to find the peak frequency?

**Comment by poojashresthacode** [comment]
We have got various peak frequencies, but you have got only one. I think it is because you have used cleaned data for calculating the frequency.

**Comment by tirthbha** [comment]
Yeah, you guys have to find out peaks for the raw data, if I were not wrong.
